src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -0,0 +1,46 @@
+
+package org.broadinstitute.dsde.agora.server.model
+
+import java.util.Date
+
+import com.novus.salat._
+import com.novus.salat.global._
+import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
+import spray.http.HttpEntity
+import spray.http.MediaTypes._
+import spray.httpx.marshalling.Marshaller
+import spray.httpx.unmarshalling.Unmarshaller
+
+import scala.annotation.meta.field
+
+object AgoraEntity {
+  implicit val AgoraEntityUnmarshaller =
+    Unmarshaller[AgoraEntity](`application/json`) {
+      case HttpEntity.NonEmpty(contentType, data) ⇒ grater[AgoraEntity].fromJSON(data.asString)
+      case HttpEntity.Empty ⇒ new AgoraEntity()
+    }
+
+  implicit val AgoraEntityMarshaller =
+    Marshaller.of[AgoraEntity](`application/json`) { (value, contentType, context) =>
+      context.marshalTo(HttpEntity(contentType, grater[AgoraEntity].toCompactJSON(value)))
+    }
+}
+
+@ApiModel(value = "Methods Query Response")
+case class AgoraEntity(@(ApiModelProperty@field)(required = true, value = "The method id")
+                       var id: Option[Int] = None,
```

Comment:
Should ID be versionId or something? I know we don't plan exposing the globally unique hash id to users, but could it cause confusion amongst devs, overloading "id"?